src/parquet/column/levels-test.cc

```diff
@@ -0,0 +1,193 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include <cstdlib>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include <gtest/gtest.h>
+
+#include "parquet/thrift/parquet_types.h"
+#include "parquet/column/levels.h"
+
+using std::string;
+
+namespace parquet_cpp {
+
+const int NUM_LEVELS = 1024;
+
+class TestLevels : public ::testing::Test {
+ public:
+  void SetUp() {
+    level_values_.resize(NUM_LEVELS);
+    output_levels_.resize(NUM_LEVELS);
+    rle_bytes_.resize(2*NUM_LEVELS);
+    bit_packed_bytes_.resize(2*NUM_LEVELS);
+  }
+
+  void TearDown() {}
+
+ protected:
+  std::vector<int16_t> level_values_;
+  std::vector<int16_t> output_levels_;
+  LevelEncoder rle_encoder_;
+  LevelDecoder rle_decoder_;
+  LevelEncoder bit_packed_encoder_;
+  LevelDecoder bit_packed_decoder_;
+  std::vector<uint8_t> rle_bytes_;
+  std::vector<uint8_t> bit_packed_bytes_;
+  int num_rle_levels_;
+  int num_bit_packed_levels_;
+};
+//test levels with bit_width = 1
+//[0,0,0,...1,1,1,..]
```

Comment:
I guess cpplint doesn't complain about this, and I'm not sure if the Google style guide provides a guideline, but I think comments are easier to read with a space between the `//` and the test. This is consistent with the rest of the codebase and more readable IMHO, would you mind terribly? Thanks